README.md

```diff
@@ -24,8 +24,8 @@ _If you cannot complete these exercises in less than 2 hours, you may find techn
 
 Write a function `isValidDecimal` that takes a `string` and returns
 
-- `true`, if it receives a string representing a valid decimal
-- `false`, if input is not a valid decimal
+- `true`, if it receives a string representing a valid decimal (e.g. `1.0`, `0`, `1`, `-1`, and `1.01`)
+- `false`, if input is not a valid decimal (e.g. `abc`, `.1`, `-.1`, and `1.`)
```

Comment:
It might be either an English or Math question. I am not quite sure if the whole numbers should be included. 

> In algebra, a decimal number can be defined as a number whose whole number part and the fractional part are separated by a decimal point.

That means a decimal number shouldn't be only a whole number.

However, in our app, we do include the whole numbers. Maybe that's why I feel puzzled.